aom.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: aom
+  version: 3.5.0
+  epoch: 0
+  description: Alliance for Open Media (AOM) AV1 codec SDK
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: BSD-2-Clause AND custom
```

Comment:
Mmh this package has a special license to control how patents are used. In the SBOM you can add a custom license ref, but I'm not sure how to handle it in the apk, which will then flow back to the SBOM but without the context.

I think we will need to do an audit of all licensing and copyright entries in the packages to ensure we are keeping the notices and expressions correct.

Reply:
Sounds good to me!  Raised an issue to track https://github.com/wolfi-dev/os/issues/340

Reply:
@puerco are you thinking we need to resolve this before merging or would the audit be enough as a followup?

Reply:
I think we can clean it up after.

Reply:
@rawlingsj no I dind't mean to hold this one. I'm sure we have a bunch of inconsistent license entries, so one more will not change that. But we need to check them soon as the number of packages grows.